CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+# v0.11.2 (2020-02-13)
```

Comment:
wrong version

Reply:
can you also add docker support for armv7 and arm64 for rabbitmq, redis, poseidon, poseidon-api, and workers images?

Reply:
Fixed.